frontend/src/components/App/PluginSettings/PluginSettings.tsx

```diff
@@ -284,15 +278,69 @@ export function PluginSettingsPure(props: PluginSettingsPureProps) {
 export default function PluginSettings() {
   const dispatch = useDispatch();
 
-  const pluginSettings = useTypedSelector(state => state.plugins.pluginSettings);
+  const pluginData = useTypedSelector(state => state.plugins.pluginData);
+
+  const oldLocalEnabledList = localStorage.getItem('headlampPluginSettings');
+  const localEnabledList = localStorage.getItem('enabledPluginsList');
+
+  let pluginsEnabledList;
+
+  /**
+   * Need to improve later, this is a temporary fix to handle the migration of the old plugin settings.
+   */
+
+  if (oldLocalEnabledList) {
+    const oldSettings = JSON.parse(oldLocalEnabledList);
+
+    const newSettings = oldSettings.map((setting: { name: string; isEnabled: boolean }) => {
```

Comment:
this `.map` doesn't seem to do that much? you could use oldSettings in the reduce directly

Reply:
fixed!